AppWidget/README.md

```diff
@@ -1,396 +1,81 @@
-    <!-- Apply padding to avoid the content of the widget colliding with the rounded corners -->
-    <item name="appWidgetPadding">16dp</item>
-</style>
-```
-
-[`values-v31/themes.xml`](app/src/main/res/values-v31/themes.xml\#L22)
-
-```xml
-<style name="Theme.AppWidget.AppWidgetContainerParent" parent="Theme.Material3.DynamicColors.DayNight">
-    <item name="appWidgetRadius">@android:dimen/system_app_widget_background_radius</item>
-    <item name="appWidgetInnerRadius">@android:dimen/system_app_widget_inner_radius</item>
-</style>
-```
-
-[`values/styles.xml`](app/src/main/res/values/styles.xml\#L18)
-
-```xml
-<style name="Widget.AppWidget.AppWidget.Container" parent="android:Widget">
-    <item name="android:id">@android:id/background</item>
-    <item name="android:background">?android:attr/colorBackground</item>
-</style>
-```
-
-[`values-v21/styles.xml`](app/src/main/res/values-v21/styles.xml\#L21)
-
-```xml
-<!--
-  Having styles for v21 because specifying android:attr/colorBackground used in the drawables
-  requires API level 21
--->
-<style name="Widget.AppWidget.AppWidget.Container" parent="android:Widget">
-    <item name="android:padding">?attr/appWidgetPadding</item>
-    <item name="android:background">@drawable/app_widget_background</item>
-</style>
-```
-
-[`drawable-v21/app_widget_background.xml`](app/src/main/res/drawable-v21/app_widget_background.xml\#L21)
-
-```xml
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
-  android:shape="rectangle">
-
-    <corners android:radius="?attr/appWidgetRadius" />
-    <solid android:color="?android:attr/colorBackground" />
-</shape>
-```
-
-[`layout/widget_checkbox_list.xml`](app/src/main/res/layout/widget_checkbox_list.xml\#L17)
-
-```xml
-layout/widget_layout.xml
-<LinearLayout
-  style="@style/Widget.AppWidget.AppWidget.Container">
-  ...
-</LinearLayout>
-```
-
-<img src="screenshots/widget_rounded_corners.png"
-    alt="screenshot for a widget with rounded corners"
-    title="screenshot for a widget with rounded corners" />
-
-## Make it easier to personalize widgets [(docs)](https://developer.android.com/about/versions/12/features/widgets#personalize-widgets)
-
-If you specify a configuration activity with the [`configure`](https://developer.android.com/reference/android/appwidget/AppWidgetProviderInfo#configure) attribute of [`appwidget-provider`](https://developer.android.com/reference/android/appwidget/AppWidgetProviderInfo), the App Widget host launches that activity immediately after a user adds the widget to their home screen.
+This sample demonstrates how to build Android AppWidgets with:
 
-Android 12 adds new options to let you provide a better configuration experience for users.
+* [`RemoteViews`](https://developer.android.com/reference/android/widget/RemoteViews)
+  and the [new Widgets APIs](https://developer.android.com/about/versions/12/features/widgets)
+  introduced in API level 31 together with
```

Comment:
General comment. Per the Google dev doc style guide (https://developers.google.com/style/product-names#android-versions), we always phrase versions using the convention "Android X (API level y)" for first use and then "Android X" thereafter. So this phrase would be "... introduced in Android 12 (API level 31) together with... "